JSON validation error on startup

```
Caused by: org.springframework.beans.BeanInstantiationException: Failed to instantiate [com.cloudesire.capri.UrbanDataServiceImpl]: Constructor threw exception; nested exception is com.fasterxml.jackson.m$dule.kotlin.MissingKotlinParameterException: Instantiation of [simple type, class com.cloudesire.capri.UrbanData] value failed for JSON property cm due to missing (therefore NULL) value for creator parame$er cm which is a non-nullable type                                                                                                                                                                            at [Source: sun.net.www.protocol.http.HttpURLConnection$HttpInputStream@2133814f; line: 1, column: 242] (through reference chain: java.util.ArrayList[0]->com.cloudesire.capri.UrbanData["cm"])                     at org.springframework.beans.BeanUtils.instantiateClass(BeanUtils.java:154) ~[spring-beans-4.3.2.RELEASE.jar!/:4.3.2.RELEASE]
        at org.springframework.beans.factory.support.SimpleInstantiationStrategy.instantiate(SimpleInstantiationStrategy.java:122) ~[spring-beans-4.3.2.RELEASE.jar!/:4.3.2.RELEASE]            
        at org.springframework.beans.factory.support.ConstructorResolver.autowireConstructor(ConstructorResolver.java:271) ~[spring-beans-4.3.2.RELEASE.jar!/:4.3.2.RELEASE]
        ... 26 common frames omitted                                                            
Caused by: com.fasterxml.jackson.module.kotlin.MissingKotlinParameterException: Instantiation of [simple type, class com.cloudesire.capri.UrbanData] value failed for JSON property cm due to missing (there$
ore NULL) value for creator parameter cm which is a non-nullable type                                                
 at [Source: sun.net.www.protocol.http.HttpURLConnection$HttpInputStream@2133814f; line: 1, column: 242] (through reference chain: java.util.ArrayList[0]->com.cloudesire.capri.UrbanData["cm"])
        at com.fasterxml.jackson.module.kotlin.KotlinValueInstantiator.createFromObjectWith(KotlinValueInstantiator.kt:53) ~[jackson-module-kotlin-2.8.1.jar!/:2.8.1]
        at com.fasterxml.jackson.databind.deser.impl.PropertyBasedCreator.build(PropertyBasedCreator.java:135) ~[jackson-databind-2.8.1.jar!/:2.8.1]
        at com.fasterxml.jackson.databind.deser.BeanDeserializer._deserializeUsingPropertyBased(BeanDeserializer.java:471) ~[jackson-databind-2.8.1.jar!/:2.8.1]
        at com.fasterxml.jackson.databind.deser.BeanDeserializerBase.deserializeFromObjectUsingNonDefault(BeanDeserializerBase.java:1194) ~[jackson-databind-2.8.1.jar!/:2.8.1]                                      at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserializeFromObject(BeanDeserializer.java:314) ~[jackson-databind-2.8.1.jar!/:2.8.1]                                                              at com.fasterxml.jackson.databind.deser.BeanDeserializer.deserialize(BeanDeserializer.java:148) ~[jackson-databind-2.8.1.jar!/:2.8.1]                                                               
        at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:277) ~[jackson-databind-2.8.1.jar!/:2.8.1]
        at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:249) ~[jackson-databind-2.8.1.jar!/:2.8.1]                                   
        at com.fasterxml.jackson.databind.deser.std.CollectionDeserializer.deserialize(CollectionDeserializer.java:26) ~[jackson-databind-2.8.1.jar!/:2.8.1]                
        at com.fasterxml.jackson.databind.ObjectMapper._readMapAndClose(ObjectMapper.java:3789) ~[jackson-databind-2.8.1.jar!/:2.8.1]                                                                      
        at com.fasterxml.jackson.databind.ObjectMapper.readValue(ObjectMapper.java:2906) ~[jackson-databind-2.8.1.jar!/:2.8.1]                                                                            
        at com.cloudesire.capri.UrbanDataServiceImpl.<init>(UrbanDataServiceImpl.kt:54) ~[classes!/:1.0.0-SNAPSHOT]  
```

--- a/server/src/main/kotlin/com/cloudesire/capri/UrbanData.kt
+++ b/server/src/main/kotlin/com/cloudesire/capri/UrbanData.kt
@@ -3,7 +3,7 @@ package com.cloudesire.capri
 import com.fasterxml.jackson.annotation.JsonSetter
 
 data class UrbanData(val nome: String, var codice: String, var zona: CodeName, var regione: CodeName,
-                     var cm: CodeName, var provincia: CodeName, var sigla: String, var codiceCatastale: String) {
+                     var provincia: CodeName, var sigla: String, var codiceCatastale: String) {
 
     var cap: List<String> = listOf()
 
@@ -15,7 +15,7 @@ data class UrbanData(val nome: String, var codice: String, var zona: CodeName, v
     }
 
     fun province(): String {
-        return if (provincia.nome != "") provincia.nome else cm.nome
+        return provincia.nome
     }
 
     fun abbreviation() = sigla

